launchcontainer/launchcontainer.py

```diff
@@ -104,65 +105,80 @@ class LaunchContainerXBlock(XBlock):
 
     @property
     def wharf_url(self, force=False):
+        """Determine which site we're on, then get the Wharf URL that said
+        site has configured."""
         site_wharf_url = None
```

Comment:
I'd probably move this down to where you try to get the actual value so it's all in the same place.